README.md

```diff
@@ -27,6 +27,7 @@ These NuGet packages provide a support for M5Stack products:
 - [M5StickC](https://docs.m5stack.com/en/core/m5stickc)
 - [M5StickCPlus](https://docs.m5stack.com/en/core/m5stickc_plus)
 - [M5Core2](https://docs.m5stack.com/en/core/core2)
+- [M5AtomLite](https://docs.m5stack.com/en/core/atom_lite)
```

Comment:
I think this should referred to as "Atom Lite" everywhere. That's the official name (no M5 prefix)
We've been doing that with all the others.
```suggestion
- [Atom Lite](https://docs.m5stack.com/en/core/atom_lite)
```